### ISSUE TITLE
Dimensions for Loss and sdf 

### ISSUE THREAD

**Issue by kabbas570** [JunMa11/SegWithDistMap#23]
Hello, @JunMa11 thanks for the nice codes,
Can you please clarify what the  (x,x,y) is for loss functions and SDF/DT? I am implementing 2D images, so I am confused about depth, height and width.

1- net_output: net logits; shape=(batch_size, class, x, y, z) --> (batch_size, class, height, width, depth)
or 
2- net_output: net logits; shape=(batch_size, class, x, y, z) --> (batch_size, class, depth,height, width)
If I consider the PyTorch required dimensions?

Thanks
Cheers
Abbas
